dockerfiles/alpine/streaming-integrator/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+
+# set base Docker image to Alpine Docker image
+FROM alpine:3.21.3
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+# install dependencies
+RUN apk add --no-cache tzdata musl-locales musl-locales-lang netcat-openbsd \
+    && rm -rf /var/cache/apk/*
+
+ENV JAVA_VERSION jdk-17.0.6+10
+
+# install Temurin OpenJDK 17
+RUN set -eux; \
+    ARCH="$(apk --print-arch)"; \
+    case "${ARCH}" in \
+       amd64|x86_64) \
+         ESUM='0df7c1a58debee2668931ba4a07cb642475b23a5c61473761b6f293eba7c024a'; \
+         BINARY_URL='https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.6%2B10/OpenJDK17U-jdk_x64_alpine-linux_hotspot_17.0.6_10.tar.gz'; \
+         ;; \
+       *) \
+         echo "Unsupported arch: ${ARCH}"; \
+         exit 1; \
+         ;; \
```

Comment:
You can add several more cases to support other OS, like arm64. SImilar to how it is done in the Dockerfiles for other OS

Reply:
When you do that change, make sure to update the relevant binary URL and the correct ESUM value

Reply:
Comparing the wso2/micro-integrator dockerfile this has the support for `aarch64|arm64) `https://github.com/poorna2152/docker-ei/blob/master/dockerfiles/alpine/micro-integrator/Dockerfile#L38. But this is for java 21 and this is not supported for java 17. https://github.com/adoptium/containers/issues/158